update to policheck-flagged text (#26014)
dotnet/docs: includes/migration-guide/runtime/wpf/resizing-grid-can-hang.md

> ### Resizing a Grid can cause it to stop responding

Reviewer: ```suggestion
### Resizing a Grid can cause the application to become unresponsive
```
